File: ThemeSwitcher/src/CODING/PYTHON/FUNCTIONAL/MAP/power.py
#  Write a Python program to create a list containing the power of said number in bases raised to the corresponding number in the index using Python map.
base = [10,20,30,40,50,60,70,80,90,100]
index = [1,2,3,4,5,6,7,8,9,10]
print("Base number and index: ")
print(base)
print(index)
result = map(pow ,base,index )
print(list(result))

File: ThemeSwitcher/src/CODING/C/REVISION/DECISION AND LOOPS/power.c
#include <stdio.h>
#include <math.h>
int main()
{
    int d, a, b;
    printf("ENTER THE BASE  : ");
    scanf("%d", &a);
    printf("ENTER THE power  : ");
    scanf("%d", &b);
    d = pow(a, b);
    printf(" %d raised to the power %d is %d ", a, b, d);
    return 0;
}